internal/gengapic/paging.go

```diff
@@ -278,6 +301,13 @@ func (g *generator) pagingCall(servName string, m *descriptor.MethodDescriptorPr
 
 func (g *generator) pagingIter(pt *iterType) {
 	p := g.printf
+	if pt.mapValueTypeName != "" {
+		p("// Holder type for string/%s map entries", pt.mapValueTypeName)
+		p("type %s struct {", pt.elemTypeName)
+		p("  Key string")
+		p("  Value %s", pt.mapValueTypeName)
```

Comment:
This is not a sufficient type name. The protobuf-go types are in a different package i.e. `computepb.Instance`. You will want to store the `*descriptor.DescriptorProto` as `pt.mapValueType` and look up the `pbinfo.NameSpec`, using the `Name` it returns for the type and the `ImportSpec.Name`.

Reply:
I went with manipulating the `mapValueType` during assingment.